pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -130,15 +151,15 @@ func (fr *FileReader) storeDashboardsInFolder(filesFoundOnDisk map[string]os.Fil
 			continue
 		}
 
-		sanityChecker.track(provisioningMetadata)
+		usageTracker.track(provisioningMetadata)
 	}
 	return nil
 }
 
 // storeDashboardsInFoldersFromFilesystemStructure saves dashboards from the filesystem on disk to the same folder
-// in Grafana as they are in on the filesystem.
+// in grafana as they are in on the filesystem
```

Comment:
Please undo this change.
```suggestion
// in Grafana as they are in on the filesystem.
```